down_frab_videos.py

```diff
@@ -254,14 +279,15 @@ def get_format_list(media_prefix):
             'From': SOURCE,
         }
 
+        print("getting ", media_prefix + "/")
```

Comment:
Any reason why you added this print statement?

Reply:
whoops. overlooked when cleaning up after debugging.

Reply:
Thought so ;).